docs/developers/completable-future.md

```diff
@@ -0,0 +1,78 @@
+
+---
+title: Using CompletableFuture
```

Comment:
...I think this topic really needs to be covered by someone else. It's going to be a bit much for Velocity to cover what is essentially basic Java concurrency primitives.